dev/arktype.io/docs/api/valueof.md

```diff
@@ -6,17 +6,14 @@ hide_table_of_contents: true
 
 ## operator
 
--   [valueOf](./valueof.md)
+-   [===](./valueof.md)
 
 ## tuple
 
--   ["===", value]
+-   ["===", value] <br/>
+-   const tupleValueOf = type(["valueOf", {a:"string"}])<br/>
 
 ## helper
 
--   valueOf(&lt;object&gt;)
-
-## example
-
--   const valueOf = type(["valueOf", {a:"string"}]) <br/>
--   const valueOf = valueOf({a:"string"}) <br/>
+-   valueOf(&lt;object&gt;) <br/>
```

Comment:
Can we replace all of these `<object>` references with `T`? In most of these cases like this, there are no constraints on the type, so saying it must be an object is misleading.